test/metabase/lib/test_metadata/graph_provider.cljc

```diff
@@ -34,25 +33,15 @@
   ;; not implemented for the simple graph metadata provider.
   nil)
 
-(defn- graph->prefix-fn [metadata-graph]
-  (let [db-name (:name (graph-database metadata-graph))]
-    (memoize
-     (fn [table-id]
-       (let [table (graph-table metadata-graph table-id)]
-         (lib.metadata.ident/table-prefix db-name table))))))
-
 (defn- graph-metadatas [metadata-graph metadata-type ids]
   (let [f (case metadata-type
             :metadata/table         graph-table
             :metadata/column        graph-field
             :metadata/segment       graph-segment
             :metadata/card          graph-card)]
     (into []
-          (comp (keep (fn [id]
-                        (f metadata-graph id)))
-                (if (= metadata-type :metadata/column)
-                  (lib.metadata.ident/attach-idents (graph->prefix-fn metadata-graph))
-                  identity))
+          (keep (fn [id]
+                  (f metadata-graph id)))
```

Comment:
Those `if`s removed feel like the most gratifying part of this PR :)

Reply:
Just in general these changes massively simplified the ident calculations in various places. Always nice to see a PR with more red lines than green.